howto/upgrade/upgrade-anbox.md

```diff
@@ -151,8 +155,51 @@ In case a reboot of the machine is required, a status message will be shown. Whe
     juju run --wait=1m lxd/0 clear-notification
 
 If the LXD charm is deployed on a machine with an NVIDIA GPU installed, by default, the NVIDIA drivers are held from being upgraded in case of downtime for all running instances due to either a manual upgrade or an [unattended-upgrade](https://wiki.debian.org/UnattendedUpgrades). The downside to this is that the machine may miss security updates for the NVIDIA drivers. To manually upgrade the NVIDIA drivers, you need to run the following Juju action:
-    
+
     juju run --wait=30m lxd/0 upgrade-gpu-drivers
+</details>
+
+
+The first step to upgrade LXD is to update the charm itself to the latest revision using:
+
+    juju refresh --channel=1.25/stable lxd
+
+Updating the charm does not include updating the LXD snap or any of the internal packages on the LXD node. After updating the charm, check which nodes have package updates available using:
+
+    juju run lxd/0 upgrade-info
+
+If you have multiple juju units for LXD nodes, make a note of all the juju units having available updates.
+
+Before proceeding make sure that all the traffic to nodes that have updates available have been drained. Run:
+
+    juju run ams/leader start-maintenance node=”<node_name>”
+
+where `<node_name>` refers to the LXD node name stored within AMS. This action marks the LXD node as unschedulable within AMS which would make sure that no new instances are spawned on the LXD node.
+
+After the node is successfully marked, run:
+
+    juju run lxd/<unit_number> --wait=30m upgrade-machine reboot=<true_or_false>
+
+where `<unit_number>` refers to the Juju unit number on each node having available updates.
+
+This action will update the gpu drivers as well as installed kernel modules which might result in a reboot being required to successfully load the new modules and drivers. This can be done automatically by setting `reboot=true` or manually at a later point in time by setting `reboot=false` and then running `juju exec lxd/<unit_number> -- sudo reboot` when required.
```

Comment:
```suggestion
This action will update the GPU drivers as well as installed kernel modules which might result in a reboot being required to successfully load the new modules and drivers. This can be done automatically by setting `reboot=true` or manually at a later point in time by setting `reboot=false` and then running `juju exec lxd/<unit_number> -- sudo reboot` when required.
```